src/modules/collection/sagas.ts

```diff
@@ -706,18 +706,19 @@ export function* collectionSaga(builder: BuilderAPI, catalyst: CatalystClient) {
       }
 
       // 6. If the collection was approved but it had a pending curation, approve the curation
-
       const newItemsCurations: ItemCuration[] = yield call(updateItemCurationsStatus, itemsToApprove, CurationStatus.APPROVED)
-      console.log('newItemsCurations: ', newItemsCurations) // TODO: Add this to the success action that will override the curations in the state
 
       // 7. Success 🎉
+      yield put(finishTPApprovalFlow(collection, itemsToApprove, newItemsCurations))
```

Comment:
Would you mind adding an assertion to the current tests to verify that this action was dispatched?